server/routes/users.js

```diff
@@ -19,10 +20,12 @@ export default () => {
     };
 
     req.auth0.users.getAll(options)
-      .then(logs => res.json(logs))
+      .then(users => res.json(users))
```

Comment:
Before returning the users, we must validate, user by user, that you have access to them.